src/about.html

```diff
@@ -75,22 +81,22 @@
               document.write('<div id="gnu-div"></div><br>');
 
               // links to third party library licenses
-              const htmlExtensions = Array('html', 'htm');
-              const licenseFolderPath = `${app.prosGlobal.rootFolder}/copyright`;
+              /*const htmlExtensions = Array('html', 'htm');
```

Comment:
This block of code is commented out, but relates to third-party licenses.  Should it actually be removed?  If so, actually remove it along with the comment line above.  If it was in error, please remove the comment block markers.